<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Primer3TmCalculatorSettingsWidget</class>
 <widget class="QWidget" name="Primer3TmCalculatorSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="toolTip">
      <string>Concentration of DNA strands</string>
     </property>
     <property name="text">
      <string>DNA Concentration</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="dsbDna">
     <property name="toolTip">
      <string>Concentration of DNA strands</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string> nM</string>
     </property>
     <property name="maximum">
      <double>9999999.990000000223517</double>
     </property>
     <property name="value">
      <double>50.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="toolTip">
      <string>Concentration of monovalent cations</string>
     </property>
     <property name="text">
      <string>Monovalent Concentration</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="dsbMonovalent">
     <property name="toolTip">
      <string>Concentration of monovalent cations</string>
     </property>
     <property name="suffix">
      <string> mM</string>
     </property>
     <property name="maximum">
      <double>99999999.989999994635582</double>
     </property>
     <property name="value">
      <double>50.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="toolTip">
      <string>Concentration of divalent cations</string>
     </property>
     <property name="text">
      <string>Divalent Concentration</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="dsbDivalent">
     <property name="toolTip">
      <string>Concentration of divalent cations</string>
     </property>
     <property name="suffix">
      <string> mM</string>
     </property>
     <property name="maximum">
      <double>9999999.990000000223517</double>
     </property>
     <property name="value">
      <double>1.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="toolTip">
      <string>Concentration of deoxynycleotide triphosphate</string>
     </property>
     <property name="text">
      <string>DNTP Concentration</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="dsbDntp">
     <property name="toolTip">
      <string>Concentration of deoxynycleotide triphosphate</string>
     </property>
     <property name="suffix">
      <string> mM</string>
     </property>
     <property name="maximum">
      <double>999999.989999999990687</double>
     </property>
     <property name="value">
      <double>0.600000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="toolTip">
      <string>Concentration of DMSO</string>
     </property>
     <property name="text">
      <string>DMSO Concentration</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="dsbDmso">
     <property name="toolTip">
      <string>Concentration of DMSO</string>
     </property>
     <property name="suffix">
      <string> %</string>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_7">
     <property name="toolTip">
      <string>Correction factor for DMSO</string>
     </property>
     <property name="text">
      <string>DMSO Factor</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="dsbDmsoFactor">
     <property name="toolTip">
      <string>Correction factor for DMSO</string>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="maximum">
      <double>9999999.990000000223517</double>
     </property>
     <property name="value">
      <double>0.600000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_8">
     <property name="toolTip">
      <string>Concentration of formamide</string>
     </property>
     <property name="text">
      <string>Formamide Concentration</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="dsbFormamide">
     <property name="toolTip">
      <string>Concentration of formamide</string>
     </property>
     <property name="suffix">
      <string> mol/l</string>
     </property>
     <property name="maximum">
      <double>9999999.990000000223517</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_9">
     <property name="toolTip">
      <string>The maximum sequence length for using the nearest neighbor model. For sequences longer than this, uses the &quot;GC%&quot; formula</string>
     </property>
     <property name="text">
      <string>NN Max Length</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="sbNnMaxLength">
     <property name="toolTip">
      <string>The maximum sequence length for using the nearest neighbor model. For sequences longer than this, uses the &quot;GC%&quot; formula</string>
     </property>
     <property name="suffix">
      <string> nt</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="value">
      <number>36</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_10">
     <property name="toolTip">
      <string>Specifies the table of thermodynamic parameters and the method of melting temperature calculation</string>
     </property>
     <property name="text">
      <string>Thermodynamic Table</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QComboBox" name="cbTable">
     <property name="toolTip">
      <string>Specifies the table of thermodynamic parameters and the method of melting temperature calculation</string>
     </property>
     <property name="currentText">
      <string notr="true">SantaLucia</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>Breslauer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SantaLucia</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_11">
     <property name="toolTip">
      <string>Specifies salt correction formula for the melting temperature calculation</string>
     </property>
     <property name="text">
      <string>Salt Correction Formula</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="cbSalt">
     <property name="toolTip">
      <string>Specifies salt correction formula for the melting temperature calculation</string>
     </property>
     <property name="currentText">
      <string notr="true">SantaLucia</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAtTop</enum>
     </property>
     <item>
      <property name="text">
       <string>Schildkraut</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SantaLucia</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Owczarzy</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
